examples/rooch_blog/sources/blog.move

```diff
@@ -0,0 +1,175 @@
+module rooch_blog::rooch_blog {
+    use std::error;
+    use std::signer;
+    use std::string::String;
+    use moveos_std::object_id::ObjectID;
+    use moveos_std::object::Object;
+    use moveos_std::storage_context::StorageContext;
+    use rooch_blog::article;
+
+    const EID_DATA_TOO_LONG: u64 = 102;
+    const EINAPPROPRIATE_VERSION: u64 = 103;
+    const ENOT_GENESIS_ACCOUNT: u64 = 105;
+
+    // === Initialize ===
+
+    // Define a function that initialize the blog app
+    fun init(storage_ctx: &mut StorageContext, account: &signer) {
+        assert!(signer::address_of(account) == @rooch_blog, error::invalid_argument(ENOT_GENESIS_ACCOUNT));
+        let _ = storage_ctx;
+        let _ = account;
+    }
+
+    // === Create ===
+
+    fun create_verify(
+        storage_ctx: &mut StorageContext,
+        account: &signer,
+        title: String,
+        body: String,
+    ): article::ArticleCreated {
+        let _ = storage_ctx;
+        let _ = account;
+        article::new_article_created(
+            title,
+            body,
+        )
+    }
```

Comment:
Remove the `verify` function, and merge it to `create` function, which can simplify the logic.